README.md

```diff
@@ -32,14 +32,22 @@ using Toopher;
 ToopherApi api = new ToopherApi("<your consumer key>", "<your consumer secret>");
 
 // Step 1 - Pair with their phone's Toopher app
-PairingStatus pairing = api.Pair("pairing phrase", "username@yourservice.com");
+// With pairing phrase
+Pairing pairing = api.Pair("username@yourservice.com", "pairing phrase");
+// With SMS
+Pairing pairing = api.Pair("username@yourservice.com", "555-555-5555");
+// With QR code
+Pairing pairing = api.Pair("username@yourservice.com");
 
 // Step 2 - Authenticate a log in
-AuthenticationStatus auth = api.Authenticate(pairing.id, "my computer");
+// With a pairing id
+AuthenticationRequest auth = api.Authenticate(pairing.id, "my computer");
+// With a username
+AuthenticationRequest auth = api.Authenticate("username", "requesterSpecifiedId");
```

Comment:
If we're trying to demonstrate different ways to use the first parameter (username / pairing id), we should use the same second parameter in these two calls to avoid confusion.